docs/docs/04-core-concepts/01-components.md

```diff
@@ -63,7 +63,7 @@ templ components can be returned from methods (functions attached to types).
 
 Go code:
 
-```
+```go
```

Comment:
In the documentation we have a special highlighter to highlight code with `templ` syntax in it.

Since this code block had some `templ` syntax I think it should be better if you used `templ` instead of `go`.

Reply:
I saw the `Go code:`, and for some reason missed the `templ` inside...